src/browser.rs

```diff
@@ -226,6 +228,46 @@ impl Browser {
             .await?;
         Ok(rx.await?)
     }
+
+    /// Create Listener for Target Changed Event
+    pub async fn target_changed_listener(&self) -> Result<EventStream<EventTargetInfoChanged>> {
+        let (tx, rx) = unbounded();
+        self.sender
+            .clone()
+            .send(HandlerMessage::AddEventListener(
+                EventListenerRequest::new::<EventTargetInfoChanged>(tx),
+            ))
+            .await?;
+
+        Ok(EventStream::new(rx))
+    }
+
+    /// Create Listener for Target Created Event
+    pub async fn target_created_listener(&self) -> Result<EventStream<EventTargetCreated>> {
+        let (tx, rx) = unbounded();
+        self.sender
+            .clone()
+            .send(HandlerMessage::AddEventListener(
+                EventListenerRequest::new::<EventTargetCreated>(tx),
+            ))
+            .await?;
+
+        Ok(EventStream::new(rx))
+    }
+
+    /// Return page of given target_id
+    pub async fn get_page(&self, target_id: TargetId) -> Result<Page> {
+        let (tx, rx) = oneshot_channel();
+        self.sender
+            .clone()
+            .send(HandlerMessage::GetPage(target_id, tx))
+            .await?;
+        match rx.await? {
```

Comment:
this is an unnecessary match block, `rx.await?` should be enough

Reply:
This part was a little hacky but I was trying to have a error for if the `target_id` isn't in targets and `HandlerMessage::GetPage` returns `None`.

Maybe I should just return `Option<Page>` instead of `Result<Page>`?